mailscanner/languages/en.php

```diff
@@ -185,6 +185,7 @@
     '15minutes03' => '15 min.:',
     'saspam03' => 'Spam',
     'sanotspam03' => 'Not spam',
+    'unknownvirusscanner03' => 'Unknown virus scanner',
```

Comment:
This should be more specific, like: 
Unknown virus scanner defined in MailScanner.conf, review your configuration and don't use 'auto' as antivirus name (see FAQ)